setup/instance/init.go

```diff
@@ -123,5 +124,31 @@ func (l *LocalInstance) createOrWriteDockerCFG(dockerInputPath string) error {
 		return err
 	}
 
+	if len(config.Auths) == 0 {
+		fmt.Println("[WARNING] Even though you have specified a path to a docker config file, " +
+			"it does not contain any auth information. If you need to add auth information " +
+			"to the docker config file, you can do so and re-run the l0-setup init command to " +
+			"include private registry authentication.\n")
+
+		fmt.Println("Press 'enter' to continue without private registry authentication: ")
+
+		var input string
+		fmt.Scanln(&input)
+	} else if config.CredsStore != "" {
+		fmt.Printf("[WARNING] Even though you have specified a path to a docker config file, "+
+			"the config file is using a credential store '%s' to cache the credentials. This "+
+			"means the credentials for private registry authentication aren't in the docker "+
+			"config file. You can either not use a credential store by removing the 'credsStore' "+
```

Comment:
This wording is a little confusing. Maybe just simplify it with "Layer0 does not support credential store authentication."